news/3191.feature

```diff
@@ -0,0 +1,2 @@
+Have pip abort if --no-binary or --only-binary receives an argument that starts
```

Comment:
```rst
Print a better error message with ``--no-binary`` is given an argument starting with ``-``.
```

The double ticks are intentional, since this is an ReST file.